tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -38,6 +44,7 @@ const globalSearchOptions = '#files-global-search-options'
 const loadingSpinner = '#files-global-search-options .loading'
 const filesViewOptionButton = '#files-view-options-btn'
 const hiddenFilesToggleButton = '//*[@data-testid="files-switch-hidden-files"]//button'
+const previewImage = `//main[@id="preview"]//div[contains(@class,"preview-player")]//img`
```

Comment:
```suggestion
const previewImage = '//main[@id="preview"]//div[contains(@class,"preview-player")]//img'
```
Just so it's consistent with others